src/helpers/utility.js

```diff
@@ -707,3 +708,45 @@ export function dedupArrObjects(arr, key) {
     return acc;
   }, []);
 }
+
+export function getMamotoTrackingSiteId() {
```

Comment:
@achen2401  another Mamoto gremlin here :) - and a call to it below.

Reply:
thank you @mcjustin! This should be fixed too.